tests/HUBViewModelBuilderTests.m

```diff
@@ -73,6 +73,17 @@ - (void)testPropertyAssignment
     XCTAssertEqualObjects(model.customData, self.builder.customData);
 }
 
+- (void)testCustomDataUpdates
+{
+    [self.builder setCustomDataValue:@"test_object" forKey:@"test_key"];
+
+    XCTAssertEqual([self.builder.customData objectForKey:@"test_key"], @"test_object");
```

Comment:
Use `XCTAssertEqualObjects` to avoid comparing against the exact same string instance.